src/webviews/apps/shared/appWithConfigBase.ts

```diff
@@ -309,13 +319,29 @@ export abstract class AppWithConfig<State extends AppStateWithConfig> extends Ap
 
 		const input = setting.querySelector<HTMLInputElement>('input[type=text], input:not([type])');
 		if (input == null) return;
-
-		const token = `\${${element.dataset.token}}`;
+		let token!: string;
+		if (
+			input.name == 'defaultDateShortFormat' ||
+			input.name == 'defaultTimeFormat' ||
+			input.name == 'defaultDateFormat'
+		) {
+			token = `${element.dataset.token}`;
+		} else {
+			token = `\${${element.dataset.token}}`;
+		}
 		let selectionStart = input.selectionStart;
 		if (selectionStart != null) {
-			input.value = `${input.value.substring(0, selectionStart)}${token}${input.value.substr(
-				input.selectionEnd ?? selectionStart,
-			)}`;
+			if (
+				input.name == 'defaultDateShortFormat' ||
```

Comment:
this is related above commit - to https://github.com/gitkraken/vscode-gitlens/pull/2505/files/09bea8d0b5ddf478cc31262e9442e0d702185bd8?diff=split&w=0#r1137539072

done -  https://github.com/gitkraken/vscode-gitlens/pull/2623/commits/4c7a5e2c106ab0f26e22be30cbbdd5e1b1a477be